### ISSUE TITLE
php artisan migrate triggers a PHP fatal error in PDOConnection.php

### ISSUE THREAD

**Issue by zrtYouyou** [BookStackApp/BookStack#2386]
Hello all,

I was trying to install BookStack for the first time following instructions found in [the official link](https://www.bookstackapp.com/docs/admin/installation/#manual) but everything is failing during the step 8.

The ```php artisan migrate``` command is giving me the 2 errors found in the following picture:
![2020-11-25_16-53-27_mintty](https://user-images.githubusercontent.com/51179979/100251254-d7353b80-2f3e-11eb-823f-0cac792d8dec.png)

Here are some informations:

- CentOS 8 (up to date)
- PHP 8.0.0
- MySQL 15.1 (with a MariaDB 10.3 server)
- Composer 2.0.7
- Bookstack user is called "bookstack" its database is "db_bookstack" and he has access permissions through localhost

Some things I tried in order to find where the problem is coming from:

- Manually create/drop a dummy table inside the bookstack database thanks to the bookstack user (it works)
- Use the MariaDB root account in the .env file (not working)
- Use quotation marks around information in the .env file (database name, mysql account...) (not working)
- Use 127.0.0.1 instead of localhost (not working)
- Disable selinux (not working)
- Disable firewalld (not working)

What can I do to prevent this error and finishing the installation process?

Because it's done on a test machine, I can provide every piece of information you'll find relevant, feel free to ask.

Have a good day!

**Comment by ssddanbrown** [comment]
Hi @zrtYouyou,
Could you try using an older version of PHP and see if that works for you? PHP7.4 would be a good choice. 

PHP8 is very very new (Officially releasing tomorrow) so Bookstack will likely not have support for it until the next release.

**Comment by zrtYouyou** [comment]
Ok so here's the follow up.

It works well with PHP 7.4, at least I can now pass the step 8.

I tried with PHP 8.0 because I had to install remi's repository in order to install just the php-tidy extension (unavailable with PHP 7.2.4, by default on CentOS).

So I was just thinking "why just enable an additional repository for one package while there is a much more recent PHP version".

I think I was a bit too greedy on this one :)

Now I have to deal with other problems (the final product is not working for the moment) but not related to this issue.

I'm closing this, thanks for the quick answer !
